keps/sig-scheduling/20200612-simplified-topology-manager.md

```diff
@@ -137,6 +139,43 @@ for _, container := range containers {
        }
 }
 ```
+## Accessing NodeResourceTopology CRD
+
+In order to allow the scheduler (deployed as a pod) to access NodeResourceTopology CRD instances, ClusterRole and ClusterRoleBinding would have to be configured as below:
+
+``` yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: exporter-handler
```

Comment:
I think we can put something related to noderesourcetopology into the name (or something similar). Exporter-handler too abstract. I understand it's just an example.

Reply:
Sure, I can update that